src/py/flwr/cli/new/new.py

```diff
@@ -125,6 +136,19 @@ def new(
 
     framework_str = framework_str.lower()
 
+    if framework_str == "flwrtune":
+        llm_challenge_value = prompt_options(
+            "Please select LLM challenge by typing in the number",
+            sorted([challenge.value for challenge in LLMChallengeName]),
+        )
+        selected_value = [
+            name
+            for name, value in vars(LLMChallengeName).items()
```

Comment:
Related to the comment above:
```suggestion
            sorted([challenge.value for challenge in LlmChallengeName]),
        )
        selected_value = [
            name
            for name, value in vars(LlmChallengeName).items()
```